sdk/core/Azure.Core/src/Shared/OperationInternal.cs

```diff
@@ -21,32 +21,25 @@ namespace Azure.Core
     /// </list>
     /// Supported members:
     /// <list type="bullet">
-    ///   <item><see cref="HasCompleted"/></item>
-    ///   <item><see cref="RawResponse"/>, used for <see cref="Operation.GetRawResponse"/></item>
-    ///   <item><see cref="UpdateStatus"/></item>
-    ///   <item><see cref="UpdateStatusAsync(CancellationToken)"/></item>
-    ///   <item><see cref="WaitForCompletionResponseAsync(CancellationToken)"/></item>
-    ///   <item><see cref="WaitForCompletionResponseAsync(TimeSpan, CancellationToken)"/></item>
+    ///   <item><see cref="OperationInternalBase.HasCompleted"/></item>
```

Comment:
Probably not important to you, since the type is `internal`, but this list won't work with the MS Docs transform;  they require at least the `<description>` child element under `<item>` to extract and transform correctly.

_(see: [docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#list))_

Reply:
Fixed